src/test/java/org/apache/commons/collections4/queue/CircularFifoQueueTest.java

```diff
@@ -421,6 +421,54 @@ public void testGetIndex() {
             assertEquals(confirmed.get(i + 2), queue.get(i));
         }
     }
+	
+	public void testAddNull() {
+        final CircularFifoQueue<E> b = new CircularFifoQueue<>(2);
+        try {
+            b.add(null);
+            fail();
+        } catch (final NullPointerException ex) {
+            assertEquals("Attempted to add null object to queue",ex.getMessage());
+            return;
+        }
+        fail();
+    }
+
+    public void testDefaultSizeAndGetError1() {
+        final CircularFifoQueue<E> fifo = new CircularFifoQueue<>();
+        assertEquals(32,fifo.maxSize());
+        fifo.add((E) "1");
+        fifo.add((E) "2");
+        fifo.add((E) "3");
+        fifo.add((E) "4");
+        fifo.add((E) "5");
+        assertEquals(5,fifo.size());
+        try {
+            fifo.get(5);
+        } catch (final NoSuchElementException ex) {
+            assertEquals("The specified index (5) is outside the available range [0, 5)",ex.getMessage());
```

Comment:
Same as my previous comment. This is a place where I would want to change the message, there is no reason why the number 5 should be in parens. Also the range syntax is left to comprehend as an exercise to the reader, which is not great IMO.

Reply:
Ok， I got it, thanks !